lib/process-services-cloud/src/lib/process/process-list/services/process-task-list-cloud.service.ts

```diff
@@ -19,13 +19,12 @@ import { Injectable } from '@angular/core';
 import { Observable, throwError } from 'rxjs';
 import { BaseCloudService } from '../../../services/base-cloud.service';
 import { map } from 'rxjs/operators';
-import { TaskListCloudServiceInterface } from '../../../services/task-list-cloud.service.interface';
 import { TaskQueryCloudRequestModel } from '../../../models/filter-cloud-model';
 import { TaskCloudNodePaging } from '../../../models/task-cloud.model';
 import { TaskListCloudSortingModel } from '../../../models/task-list-sorting.model';
 
 @Injectable({ providedIn: 'root' })
-export class ProcessTaskListCloudService extends BaseCloudService implements TaskListCloudServiceInterface {
```

Comment:
`ProcessTaskListCloudService ` shouldn't implement that interface - it's not event shared between tasks and processes.